src/hooks/screen/useChatScreenHook.ts

```diff
@@ -147,7 +195,7 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
       if (e?.response?.data?.status === 'Channel is blocked') return;
 
       setTimeout(() => {
-        sendChat(message, iteration + 1, currentChatSchema).catch((sendChatError) => {
+        sendChat(message, attachments, iteration + 1, currentChatSchema).catch((sendChatError) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sendChat` function call inside the `setTimeout` has been updated to include the `attachments` parameter. This is a necessary change to ensure that attachments are included when retrying to send a chat message after a failure.
```diff
+         sendChat(message, attachments, iteration + 1, currentChatSchema).catch((sendChatError) => {
           console.log(sendChatError);
         });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->